src/sass/app.scss

```diff
@@ -146,7 +148,7 @@ a {
 
 .text-area {
 	text-align: left;
-	width: calc(100% - #{$width-image-container});
+	@include flex(calc(100% - #{$width-image-container}));
```

Comment:
I think I didn't make my point clear enough, what I meant here is to use only flexbox approach and avoid the unnecessary **calc()** CSS function and use only flexbox values here: `flex: 1 1 100%;` that'll do the work just fine :)
![image](https://user-images.githubusercontent.com/662930/38462911-cd339d34-3ab3-11e8-95cd-d59087a6ca92.png)

Reply:
Solved in commit d6f47ca